src/main/cli/cli.c

```diff
@@ -164,6 +164,7 @@ bool cliMode = false;
 #include "sensors/compass.h"
 #include "sensors/esc_sensor.h"
 #include "sensors/gyro.h"
+#include "sensors/gyro_init.h"
```

Comment:
We could potentially simplify this a bit by keeping the API for the combination of `gyro.c` and `gyro_init.c` in `gyro.h` - but not sure if this is the right thing to do.